.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+on:
+  push:
```

Comment:
`push:` should match the default behavior outlined by CircleCI [here](https://circleci.com/docs/triggers-overview/#run-a-pipeline-on-commit-to-your-code-repository).

> Once a project is hooked up to CircleCI, a pipeline is triggered each time a commit is pushed on a branch that has a `.circleci/config.yml` file included.

Reply:
Nit, but not required - I don't think we need the colon after the push

Reply:
Addressed in [`f402406` (#223)](https://github.com/panther-labs/panther_analysis_tool/pull/223/commits/f40240698bc222edd382eb215d41f6c2676119f7).